setup/configs/nginx-dhis2.conf

```diff
@@ -1,164 +1,59 @@
-# connection limit zones
-limit_conn_zone $server_name zone=dvsconn:10m;
-# rate limit zones
-limit_req_zone $binary_remote_addr zone=ip:10m rate=4r/m;
-limit_req_zone $server_name zone=all:10m rate=20r/m;
+user   www-data;
+worker_processes  auto;
 
-# performance log format
-log_format performance '$remote_addr\t[$time_local]\t-\t$remote_user\t'
```

Comment:
Can we keep the performance log format?  It is useful for analysis

Reply:
You are right. I have fixed and uploaded the log format